lib/web/helper.ex

```diff
@@ -4,7 +4,6 @@ defmodule Beaker.Web.Helper do
   @colors ~w(yellow orange red magenta violet blue cyan green)
 
   def random_color do
-    :random.seed(:os.timestamp)
```

Comment:
As the rand module descries  http://erlang.org/doc/man/rand.html: 

```
The following algorithms are provided:

exsplus
Xorshift116+, 58 bits precision and period of 2^116-1

exs64
Xorshift64*, 64 bits precision and a period of 2^64-1

exs1024
Xorshift1024*, 64 bits precision and a period of 2^1024-1
The default algorithm is exsplus.If a specific algorithm is required, ensure to always use seed/1 to initialize the state
```
I think the precision is enough here.